sms_toolkit/constants/unicode_to_gsm_map.py

```diff
@@ -137,3 +139,273 @@
     0x03A9: [0x15],
     0x20AC: [0x1B, 0x65]
 }
+
+# Map that can be used to replace unicode characters with a GSM equivalent.
```

Comment:
Should we post a link for the source of these here?

Reply:
Yup meant to do that just forgot. Thanks for the callout!